### DISCUSSION TITLE
The MSS310 LED is behaving differently with HA compared to when using the Meross app.

### DISCUSSION THREAD

**Discussion by hakevl** [krahabb/meross_lan#519] [Q&A]
When I toggle the power state of a MSS310 plug with the Meross app, the LED also toggles. However, when I toggle the power state with HA (using this integration), it keeps the LED lit. Am I doing something wrong, or has something changed that this is no longer working? I think it used to work in the past.

**Comment by krahabb** [comment]
This looks like the issue being tracked in #305...still no known resolution but I'll try to get back to it when I'll start the new 'development cycle'
